capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -0,0 +1,21 @@
+document.body.prepend(dynamicButton);
+
+function createClassroom() {
+    var auth2 = gapi.auth2.getAuthInstance();
+    var name = "";
+    var email = "";
+    if (auth2.isSignedIn.get()) {
+        var name = auth2.currentUser.get().getBasicProfile().getName();
+        var email = auth2.currentUser.get().getBasicProfile().getEmail();
+    }
+
+    var subject = document.querySelector('#subject').value;
+    var classroomData = JSON.stringify({ "nickname": name, "userId": email, "subject": subject });
```

Comment:
"userId: email" seems kind of confusing

Reply:
Yeah, for now I'm using their email as their ID since I don't to pass their user ID to the backend.